draft-ietf-capport-architecture.xml

```diff
@@ -136,10 +136,9 @@
          <t>Solutions MUST operate at the layer of Internet Protocol (IP) or
             above, not being specific to any particular access technology such
             as Cable, WiFi or 3GPP.</t>
-         <t>Solutions SHOULD allow a device to be alerted that it is in a
+         <t>Solutions MAY allow a device to be alerted that it is in a
```

Comment:
Should this instead be:

"Solutions MAY alert devices that they are in a captive network..."

The solution is not really "allowing" the the device to be alerted—the network is just alerting or not.